modules/getting-started/proc-customize-rhdh-homepage.adoc

```diff
@@ -22,48 +21,72 @@ proxy:
-====
-
-You can use the https://github.com/redhat-developer/red-hat-developer-hub-customization-provider[`red-hat-developer-hub-customization-provider`] as an example service, which provides data for both Home page and Tech Radar. The `red-hat-developer-hub-customization-provider` service provides the same data as default {product-short} data. You can fork the `red-hat-developer-hub-customization-provider` service repository from GitHub and modify it with your own data, if required.
---
 
-This section describes how you can deploy the `red-hat-developer-hub-customization-provider` service onto the cluster where the {product-short} Helm Chart is deployed.
+* A separate service that provides the Home page data in JSON format using an API. When using a separate service, you can do the following:
+** Use the same service to provide the data to all configurable {product-short} pages or use a different service for each page.
+** Use the https://github.com/redhat-developer/red-hat-developer-hub-customization-provider[`red-hat-developer-hub-customization-provider`] as an example service, which provides data for both the Home and Tech Radar pages. The `red-hat-developer-hub-customization-provider` service provides the same data as default {product-short} data. You can fork the `red-hat-developer-hub-customization-provider` service repository from GitHub and modify it with your own data, if required.
+** Deploy the `red-hat-developer-hub-customization-provider` service and the {product-short} Helm chart on the same cluster.
 
 .Prerequisites
 
-* You have installed the {product} using Helm Chart. For more information, see xref:proc-install-rhdh_rhdh-getting-started[].
+* You have installed the {product} using the Helm chart. For more information, see xref:proc-install-rhdh_rhdh-getting-started[].
 
 .Procedure
 
-. In Red Hat OpenShift, select *+Add* and click *Import from Git* option.
-. Add the URL of your Git repository to the *Git Repo URL* field.
+To use a separate service to provide the Home page data, complete the following steps:
+
+. From the *Developer* perspective in the {ocp-brand-name} web console, click *+Add* > *Import from Git*.
+. Enter the URL of your Git repository into the *Git Repo URL* field.
 +
 --
-To use the `red-hat-developer-hub-customization-provider` service, you can add the URL of https://github.com/redhat-developer/red-hat-developer-hub-customization-provider[red-hat-developer-hub-customization-provider] repository.
+To use the `red-hat-developer-hub-customization-provider` service, add the URL for the https://github.com/redhat-developer/red-hat-developer-hub-customization-provider[red-hat-developer-hub-customization-provider] repository.
 --
 
-. In the *General* section, rename the value in the *Name* field to *rhdh-customization-provider* and click *Create*.
-. Go to the *Advanced Options* and copy the value from the *Target Port*.
+[NOTE]
+====
+The `rhdh-customization-provider` service contains the 8080 port.
```

Comment:
```suggestion
The `rhdh-customization-provider` service exposes port 8080 by default. You can override it using the `PORT` environment variable.
```

Reply:
I guess following up on the [comment](https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/368/files#r1674263631) below, this would probably need to mention that adding a custom `PORT` would also require configuring the `Target Port` to match.